samples/embedded-login/src/main.js

```diff
@@ -13,14 +13,14 @@ import * as forgerock from '@forgerock/javascript-sdk';
 const FATAL = 'Fatal';
 forgerock.Config.set({
   clientId: process.env.WEB_OAUTH_CLIENT, // e.g. 'ForgeRockSDKClient'
-  redirectUri: `${window.location.origin}/callback`, // e.g. 'https://sdkapp.example.com:8443/_callback'
-  scope: process.env.SCOPE, // e.g. 'openid profile me.read'
+  redirectUri: `${window.location.origin}/callback`, // e.g. 'https://sdkapp.example.com:8443/callback'
+  scope: process.env.SCOPE, // e.g. 'openid profile email address phone me.read'
```

Comment:
It looks like we don't recommend the `me.read` scope any more in our core tutorial docs: https://backstage.forgerock.com/docs/sdks/latest/serverconfiguration/cloud/create-oauth2-client.html.

Reply:
Yeah I left that alone in case it was needed by the samples, but if not let's pull that too.